lnrpc-ideal/invoicesrpc/invoices.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
```

Comment:
Perhaps worth having a look at our current `List{Payments/Invoices}` api in this as well? Even if it's just to compare how similar they are/ anything that's currently missing?